modules/con_ipi-installation-workflow.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_bare_metal/ipi/ipi-install-prerequisites.adoc
+
+:_mod-docs-content-type: CONCEPT
+
+[id="installation-workflow_{context}"]
+= Installation workflow
+
+The installation program supports interactive mode, but Red Hat recommends preparing an `install-config.yaml` file in advance, containing the provisioning details for all of the bare-metal hosts.
+
+Add the relevant cluster details to the `install-config.yaml` file. The installation program ships the `install-config.yaml` file. The administrator generates the manifests and verifies all prerequisites.
```

Comment:
```suggestion
Add the relevant cluster details to the `install-config.yaml` file. The installation program ships the `install-config.yaml` file. The administrator generates the manifests and verifies all prerequisites.
```

I'm not sure what "ships" means in this context. Is there a more descriptive or accurate way to phrase this?

Reply:
Updated as suggested.